modules/ocf/manifests/ssl/lets_encrypt/dns.pp

```diff
@@ -23,7 +25,7 @@
     # This exec can be notified to get it to run dehydrated again, even if the
     # cert will not expire soon.
     command     => '/usr/bin/dehydrated --cron --privkey /etc/ssl/lets-encrypt/le-account.key',
-    user        => ocfletsencrypt,
+    user        => $owner,
```

Comment:
You're sure this is safe to run as root, right?

Reply:
This should be, but it's not great, I think we can just change this to use the constructed bundle made by concat and that'll fix all of this cert stuff essentially, since the bundle file at least has the correct permissions.